ProvinceMapper/Source/Provinces/Province.cpp

```diff
@@ -0,0 +1,30 @@
+#include "Province.h"
+
+Pixel::Pixel(int tx, int ty, unsigned char tr, unsigned char tg, unsigned char tb): x(tx), y(ty), r(tr), g(tg), b(tb)
+{
+}
+
+bool Province::operator==(const Province& rhs) const
+{
+	return ID == rhs.ID;
+}
+
+bool Province::operator==(const Pixel& rhs) const
+{
+	return r == rhs.r && g == rhs.g && b == rhs.b;
+}
+
+bool Province::operator!=(const Province& rhs) const
+{
+	return ID != rhs.ID;
+}
+
+bool Province::operator<(const Province& rhs) const
+{
+	return ID < rhs.ID;
+}
+
+bool Province::operator!() const
```

Comment:
If this was used anywhere, I missed it. In any case, I dislike the idea you could have an incomplete province existing. If not fully formed, it should simply not exist.

Or if this is indicating something else, it can use a better name than an operator overload.